### PR TITLE
docs(components): [overview] add keyboard control selection component

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

After searching by keyword, use the tab key to quickly select and jump.

play

### CODE REVIEW

**Comment by kooriookami** [comment]
image

Do you think outline is better?
```
outline: 2px solid #409eff;
outline-offset: 1px;
```

File: docs/.vitepress/vitepress/components/globals/overview.vue [hunk 1]
```diff
@@ -25,6 +25,8 @@
           <el-card
             v-for="(item, index) in group.children"
             :key="index"
+            tabindex="0"
+            :data-link="item.link"
             shadow="hover"
             @click="toPage(item.link)"
           >
```
Comment (kooriookami) [inline docs/.vitepress/vitepress/components/globals/overview.vue:32]:
```suggestion
          <el-card
            v-for="(item, index) in group.children"
            :key="index"
            tabindex="0"
            shadow="hover"
            @click="toPage(item.link)"
            @keydown.enter="toPage(item.link)"
          >
```
You don't need to write complicated.

File: docs/.vitepress/vitepress/components/globals/overview.vue [hunk 2]
```diff
@@ -104,6 +106,21 @@ const getIcon = (link: string) => {
...
+      toPage((firstCard as HTMLElement).dataset.link!)
+    }
+  }
+}
+onMounted(() => {
+  document.addEventListener('keydown', handleKeydown)
+})
+onBeforeUnmount(() => {
+  document.removeEventListener('keydown', handleKeydown)
+})
```
Comment (kooriookami) [inline docs/.vitepress/vitepress/components/globals/overview.vue:123]:
```suggestion
```

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: docs/.vitepress/vitepress/components/globals/overview.vue (hunk 1 above)
Comment (btea) [reply docs/.vitepress/vitepress/components/globals/overview.vue:32]: Makes sense. 👍

**Comment by btea** [comment]
> Do you think outline is better?
> 
> ```
> outline: 2px solid #409eff;
> outline-offset: 1px;
> ```

image

The colors could use some tweaking, but the shadows look better.

**Comment by kooriookami** [comment]
Let me ask the designer~

**Comment by kooriookami** [comment]
https://github.com/element-plus/element-plus/assets/38392315/aabfa471-9d53-4ab6-bbf6-ada7fba7ba4f

There's a black border shows before outline.

**Comment by btea** [comment]
It should be caused by the transition effect set by `el-card`.

**Review by kooriookami — APPROVED** [review]

**Pull request merged**